src/main/java/org/folio/circulation/domain/OpeningDay.java

```diff
@@ -33,16 +34,24 @@ static OpeningDay createClosedDay() {
   private LocalDate date;
   private boolean allDay;
   private boolean open;
+  private boolean exceptional;
 
-  OpeningDay(JsonObject jsonObject, String key) {
-    JsonObject openingDayJson = jsonObject.getJsonObject(key);
+  OpeningDay(JsonObject openingDayJson) {
+    this.allDay = openingDayJson.getBoolean(ALL_DAY_KEY, false);
+    this.open = openingDayJson.getBoolean(OPEN_KEY, false);
+    this.openingHour = fillOpeningDay(openingDayJson);
     String dateProperty = openingDayJson.getString(DATE_KEY);
     if (dateProperty != null) {
       this.date = LocalDate.parse(dateProperty, DATE_TIME_FORMATTER);
     }
-    this.allDay = openingDayJson.getBoolean(ALL_DAY_KEY, false);
-    this.open = openingDayJson.getBoolean(OPEN_KEY, false);
-    this.openingHour = fillOpeningDay(openingDayJson);
+    Boolean exceptionalProperty = openingDayJson.getBoolean(EXCEPTIONAL_KEY);
```

Comment:
What is an exceptional day?

Reply:
It means that the day is an exception to the regular schedule. An exceptional day overrides an ordinary day in the calendar.

Reply:
Ok, thank you